backend/de.metas.serviceprovider/src/main/sql/postgresql/system/45-de.metas.serviceprovider/5653750_sys_gh13430_S_Issue_display_ExternallyUpdatedAt_in grid_and_filter.sql

```diff
@@ -0,0 +1,113 @@
+-- Column: S_Issue.ExternallyUpdatedAt
```

Comment:
it's fine for now but pls avoid spaces in SQL script filenames.

Reply:
removed the space from filename